src/test/groovy/org/osiam/test/integration/ControllerIT.groovy

```diff
@@ -319,34 +318,21 @@ class ControllerIT extends AbstractIT {
     def 'OSNG-467: Deactivating a user should revoke his access token'() {
         given: 'active user with valid access token'
         def userId = "cef9452e-00a9-4cec-a086-d171374ffbef"
-        AccessToken serviceAccessToken = OSIAM_CONNECTOR.retrieveAccessToken(Scope.ADMIN)
-        UpdateUser updateUser = new UpdateUser.Builder().updateActive(false).build()
+        def serviceAccessToken = OSIAM_CONNECTOR.retrieveAccessToken(Scope.ADMIN)
+        def user = OSIAM_CONNECTOR.getUser(userId, serviceAccessToken)
+        def updateUser = new User.Builder(user).setActive(false).build()
```

Comment:
Please move the initialization down into the 'when' block. It makes it easier to understand.